.github/workflows/main.yml

```
@@ -1,29 +1,21 @@
 # https://help.github.com/en/github/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
 # https://github.com/marketplace/actions/gradle-command
-name: Run unit tests on each commit
-on:
-  pull_request:
-    paths:
-      - '!**.md'
-      - '!.editorconfig'
-      - '!**/.gitignore'
-      - '!**.adoc'
-  push:
-    paths:
-      - '!**.md'
-      - '!.editorconfig'
-      - '!**/.gitignore'
-      - '!**.adoc'
+name: CI unit tests
+on: [pull_request]
 jobs:
   unittests:
     runs-on: ubuntu-latest    
     steps:
       - uses: actions/checkout@v1
-      - name: Run a one-line script
-        run: echo Hello, world!
       - uses: actions/setup-java@v1
         with:
           java-version: 8
       - uses: eskatos/gradle-command-action@v1
         with:
           arguments: testDebugUnitTest
```

You miss an `id` on the step that runs Gradle so the build scan output can be referenced in the commenting step:
```suggestion
          arguments: testDebugUnitTest
          id: gradle
```
See https://github.com/eskatos/gradle-command-action/issues/10#issuecomment-626306465